lnd_test.go

```diff
@@ -8158,7 +8158,7 @@ func testAsyncPayments(net *lntest.NetworkHarness, t *harnessTest) {
 	// Open up a payment stream to Alice that we'll use to send payment to
 	// Bob. We also create a small helper function to send payments to Bob,
 	// consuming the payment hashes we generated above.
-	ctxt, _ = context.WithTimeout(ctxb, time.Minute)
+	ctxt, _ = context.WithTimeout(ctxb, 3*time.Minute)
```

Comment:
What change now requires this long a time out?

Reply:
This was initially done to combat the added propagation delay. But after the trickle delay is customisable, this might not be needed anymore. I will remove this commit.